include/vrv/iomusxml.h

```diff
@@ -92,6 +92,20 @@ namespace musicxml {
         std::string m_endID;
     };
 
+    class EndingInfo {
+    public:
+        EndingInfo(std::string endingNumber, std::string endingType, std::string endingText)
```

Comment:
change parameters to `const std::string &`

Reply:
Ok. But should we do the same also for the neighboring classes (`OpenHairpin`, `OpenSlur`, `CloseSlur`, ...)?

Reply:
Yes, this would be good.

Reply:
Done.